src/i965_avc_encoder.c

```diff
@@ -11231,7 +11231,7 @@ gen9_avc_vme_context_init(VADriverContextP ctx, struct intel_encoder_context *en
         avc_state->kernel_trellis_enable = 1;
         avc_state->lambda_table_enable = 1;
         avc_state->brc_split_enable = 1;
-        avc_state->adaptive_transform_decision_enable = 1;// CNL
+        avc_state->adaptive_transform_decision_enable = 0;
```

Comment:
This line can be removed as the avc_state->adaptive_transform_decision_enable has been set to 0 by default.

Reply:
Better fix in https://github.com/01org/intel-vaapi-driver/issues/323. And remove this line.